### DISCUSSION TITLE
wayland mode can't run in prime offload

### DISCUSSION THREAD

**Discussion by Rsplwe** [ppy/osu#24590] [Q&A] [answered]
### Type

Crash to desktop

### Bug description

OS: Arch Linux 6.4.10-arch1-1
NVIDIA: 535.98
Mesa: 23.1.5

wayland mode can't run in prime offload

start command:
`__NV_PRIME_RENDER_OFFLOAD=1 __VK_LAYER_NV_optimus=NVIDIA_only __GLX_VENDOR_LIBRARY_NAME=nvidia SDL_VIDEODRIVER=wayland ./osu-lazer`

### Version

2023.815.0

### Logs

Console:
```
[destroyed object]: error 7: importing the supplied dmabufs failed
warning: queue 0x55ee58b83d90 destroyed while proxies still attached:
  wl_buffer@43 still attached
warning: queue 0x55ee58b75f00 destroyed while proxies still attached:
  zwp_linux_dmabuf_feedback_v1@40 still attached
  zwp_linux_dmabuf_feedback_v1@33 still attached
  zwp_linux_dmabuf_v1@34 still attached
```
runtime.log
```
2023-08-18 19:10:10 [verbose]: SDL error log [debug]: unable to show color buffer in an OS-native window (call to eglSwapBuffers failed, reporting an error of EGL_BAD_SURFACE)
```

**Comment by bdach** [comment, marked as answer]
You're going to have to pick another upstream to report this to (your distro? your WM? SDL? nvidia - good luck with that?) Or just stop messing with nonstandard configs.

We can't do anything with this error here.
